guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc

```diff
@@ -56,7 +56,7 @@ Optional: If the *Registration* feature is not enabled on your {SmartProxy}, ent
 
 . From the *Operating System* list, select the operating system of hosts that you want to register.
 
-. Optional: If you want to register hosts through {SmartProxy}, from the *{SmartProxy}* list, select the {SmartProxy} to register hosts to.
+. From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through {SmartProxy}.
```

Comment:
Isn't it an optional step? Users need to do this only in the case when they want to register through Capsule, isn't it? I would keep the `Optional:` part in this step to avoid confusion.

Reply:
IIUC, the internal Capsule will be selected if they do not select any Capsule, and that can cause problems in some cases e.g. where they connected using an IP not a FQDN

Reply:
Oh, so one can select the internal Capsule. Yup, then this field is always required.

Reply:
```suggestion
. From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through. You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
```
How about making it explicit like that?

Reply:
I think lstejska wants to make it default to that setting in the web UI, not just in the code, so that it is clear what will happen if you do nothing. In which case "You must" would be a little strong. Lets wait now for lstejska to give feedback.

Thank you

Reply:
> that can cause problems in some cases e.g. where they connected using an IP not a FQDN.

To be honest we didn't count with the users that connect to Satellite via IP and not a FQDN,
there is an open issue for it: https://projects.theforeman.org/issues/31821

So my advice would be to select Capsule every time, just in case.

Reply:
Thank you